packages/image_picker/image_picker_ios/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: image_picker_ios
 description: iOS implementation of the video_picker plugin.
 repository: https://github.com/flutter/plugins/tree/main/packages/image_picker/image_picker_ios
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+image_picker%22
-version: 0.8.5
+version: 0.8.6
```

Comment:
This is a bugfix of a pre-1.0 package, so should update the build number instead. So this should be 0.8.5+1.